portfolio/src/main/webapp/script.js

```diff
@@ -25,4 +25,28 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+}
+
+function getData() {
+    console.log('Fetching data.');
+    const responsePromise = fetch('/random-quote');
+    responsePromise.then(handleResponse);
+}
+
+function handleResponse (response) {
+    const textPromise = response.text();
+    textPromise.then(addQuoteToDom);
+}
+
+function addDataToDom(quote) {
+  console.log('Adding data to dom: ' + data);
+
+  const dataContainer = document.getElementById('data-container');
+  dataContainer.innerText = data;
+}
+
+function getDataUsingArrowFunctions() {
```

Comment:
`UsingArrowFunctions` is more of an implementation detail. I'd suggest a rename of this if you want. Callers shouldn't know anything about how a function is internally implemented.